### ISSUE TITLE
I am not receiving any data

### ISSUE THREAD

**Issue by hem0773** [roema/Open-Glider-Network-Groundstation#25]
I am not receiving any data

Can this be due to my TTgo-Beam V1.1 which may have a Lora chip that is not supported

**Comment by roema** [comment]
Where do you come from? In Europe 868 is used.

**Comment by hem0773** [comment]
yes of course Europe!
I thought that the software might only work with the SX1262 and not with the SX1276 ?!

**Comment by roema** [comment]
No, it works with both, 1262 and 1276.

Connect you ttgo device to arduino ide and start serial monitor.
On bootup you can see, wich chip is found, or if no chip is found.

**Comment by hem0773** [comment]
OGNB-ESP32 FW.REV: 0.1.0-24 DEV.ID: f2463208
Copyright (C) 2020 Manuel Roesel. All rights reserved.

Reset reason: 0
POWERON_RESET
Free heap size: 255228
Vbat power on reset

EEPROM version: 94
[D][WiFiGeneric.cpp:374] _eventCallback(): Event: 0 - WIFI_READY
[D][WiFiGeneric.cpp:374] _eventCallback(): Event: 2 - STA_START
WiFi config changed.
.6.6.6.6.6[D][WiFiGeneric.cpp:374] _eventCallback(): Event: 4 - STA_CONNECTED
[D][WiFiGeneric.cpp:374] _eventCallback(): Event: 7 - STA_GOT_IP
[D][WiFiGeneric.cpp:419] _eventCallback(): STA IP: 192.168.178.60, MASK: 255.255.255.0, GW: 192.168.178.1
IP address: 192.168.178.60
Hostname: OGNB-463208
UDP server has started at port: 12389
SX1276 RFIC is detected.
INFO: TTGO T-Beam rev. 08 is detected.
Flash memory ID: 684016
INFO: GNSS module HW version: 00040007
INFO: GNSS module FW version: 7.03 (45969)
[I][ArduinoOTA.cpp:141] begin(): OTA server at: OGNB-463208.local:3232
[W][SPIFFS.cpp:71] begin(): SPIFFS Already Mounted!
Starting NTP UDP
Local port: 2390
#1 NTP server's IP address: 194.25.134.196
sending NTP packet...
Reply packet received, length=48
Seconds since Jan 1 1900 = 3844604739
Unix time = 1635615939
The UTC time is 17:45:39
[E][WiFiUdp.cpp:183] endPacket(): could not send data: 12
[I][ping.cpp:324] ping_start(): PING 51.68.189.96: 32 data bytes

[D][ping.cpp:222] ping_recv(): 60 bytes from 51.68.189.96: icmp_seq=1 time=25.544 ms

[D][ping.cpp:222] ping_recv(): 60 bytes from 51.68.189.96: icmp_seq=2 time=22.081 ms

[I][ping.cpp:344] ping_start(): 2 packets transmitted, 2 packets received, 0.0% packet loss

[I][ping.cpp:349] ping_start(): round-trip min/avg/max/stddev = 22.081/23.812/25.544/1.732 ms

[E][WiFiUdp.cpp:183] endPacket(): could not send data: 12
[E][WiFiUdp.cpp:183] endPacket(): could not send data: 12

**Comment by roema** [comment]
Hey hem!

Looks good, SX1276 is detected. 
Do you have 868 MHZ or 915 MHZ version?

**Comment by hem0773** [comment]
I can't get any further and have ordered a new board

get in touch

**Comment by hem0773** [comment]
Reception is quite bad compared to the SDR and Raspberry

**Comment by roema** [comment]
Hi @hem0773 !

We did a lot of tests and were able to gain some knowledge. What leads to a significant deterioration in the reception range is the presence of cell phone stations in the immediate vicinity.

In open terrain on a hill with good reception properties, we were able to detect up to 130km at FLARM. Flarm is FSK and not an optimal protocol for the SX chipset. FANET should deliver far better results because it is based on LORA.

From today's perspective, the ESP32 ground station is well suited for applications that are supposed to function in a power-saving manner. The new binary protocol is a lot better than APRS and also allows sending via lora wan without much traffic.

Regarding APRS, it would be high time to replace this with an efficient protocol.

Regards, Manuel
